assets/templates/snippets/snippet_userdisplay.html

```diff
@@ -0,0 +1,17 @@
+{% load static %}
+
+<span class="d-flex justify-content-between align-items-center">
+    <span>{{ entry.name }}</span>
+
+    <span class="d-flex align-items-right">
```

Comment:
`align-items-right` is not an existing css class as far as I know. You can remove both classes here I think, just leave `<span>` without the class argument

Reply:
Nope, it wasn't. align-items-center was, so I thought align-items-right did as well. Funny enough, removing align-items-center resulted in a default behaviour that I wanted, so I didn't notice it XD